activemodel/lib/active_model/validations/clusivity.rb

```diff
@@ -11,7 +11,7 @@ module Clusivity # :nodoc:
       ERROR_MESSAGE = "An object with the method #include? or a proc, lambda or symbol is required, " \
                       "and must be supplied as the :in (or :within) option of the configuration hash"
 
-      def check_validity!
+      def check_clusivity_validity!
```

Comment:
We could keep this as `check_validity!` and change the call in `NumericalityValidator` to `super`.  What do you think?